backend/danswer/db/engine.py

```diff
@@ -27,6 +28,15 @@ def get_db_current_time(db_session: Session) -> datetime:
     return result
 
 
+def translate_db_time_to_server_time(
+    db_time: datetime, db_session: Session
+) -> datetime:
+    server_now = datetime.now()
```

Comment:
should this be `datetime.now(timezone.utc)`?

Reply:
This one is timezone unaware meaning it's like ms from epoch, I think it's better to keep everything untimezoned if we can?